builtin/logical/pki/path_resign_crls.go

```diff
@@ -26,6 +30,12 @@ const (
 	formatParam             = "format"
 )
 
+var (
+	akOid       = asn1.ObjectIdentifier{2, 5, 29, 20}
+	crlNumOid   = asn1.ObjectIdentifier{2, 5, 29, 20}
```

Comment:
typo? This is the same as the one above it.

Reply:
*sigh* copy/paste fail, fixed and added a test to verify constants match expected values...